packages/kbn-config-schema/src/types/stream_type.test.ts

```diff
@@ -51,6 +51,7 @@ describe('#defaultValue', () => {
         "_events": Object {},
         "_eventsCount": 0,
         "_maxListeners": undefined,
+        Symbol(kCapture): false,
```

Comment:
This snapshot change is caused by a fix in the internal deep copy function `joi` was using. Symbols are now properly copied.

Reply:
Maybe we should refactor the test to assert against `Stream` instance?